documento-in-consultazione/definizioni.rst

```diff
@@ -188,7 +188,7 @@ Tabella 9: esempio di requisito di qualità
 +------------------+-------------------------------------------------------------------------------------------------------+
 | **ID Requisito** | **Descrizione**                                                                                       |
 +==================+=======================================================================================================+
-| RQ1              | Il tempo di risposta del sistema all'inserimento della password utente deve essere inferiore a 10 sec |
+| RQ1              | Il tempo di risposta del sistema (90° pecentile) all'inserimento della password utente deve essere inferiore a 10 sec. Ovvero, il 90 percento delle richieste di inserimento password devono rispondere all'utente entro 10 secondi. |
```

Comment:
Probabile errore di battitura: "pecentile" invece di "percentile"; si raccomanda inoltre l'uso dell'[indicatore ordinale](https://it.wikipedia.org/wiki/Indicatore_ordinale) (º) in luogo del simbolo di [grado](https://it.wikipedia.org/wiki/Grado_(simbolo)) (°).

Reply:
Grazie!